config.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Canonical Ltd.
+# Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
 
 options:
```

Comment:
I would update the description of extra-user-role, with similar texts as the one provided in the readme, that was a bit clearer and with examples